public/fullnodes.json

```diff
@@ -4,6 +4,7 @@
       "http://165.232.136.149:8080",
```

Comment:
Nit, but it would be nice if the list only appears once.

Reply:
yes, this is deprecated